bundles/org.eclipse.e4.ui.workbench/src/org/eclipse/e4/ui/internal/workbench/ExtensionTrackeContextFunction.java

```diff
@@ -36,12 +36,15 @@ public class ExtensionTrackeContextFunction extends ContextFunction implements E
 	@Reference
 	private ILog log;
 
-	private Map<IEclipseContext, IExtensionTracker> createdObjects = new ConcurrentHashMap<>();
+	private Map<IEclipseContext, IExtensionTracker> createdObjects = null;
```

Comment:
Why lazy init this? any advantage? THis wil possibly introduce race contions?

Reply:
No reason and no advantage at all.. just to initialize with the tracker... this is the same for me... what do you mean by race contions ?